examples/go-tc-counter/main.go

```diff
@@ -139,7 +139,7 @@ func main() {
 		}
 
 		// 3. Get access to our map
-		mapPath, err = configMgmt.RetrieveMapPinPath(ctx, c, paramData, bpfdHelpers.Tc.Uint32(), "tc_stats_map");
+		mapPath, err = configMgmt.RetrieveMapPinPath(ctx, c, paramData, bpfdHelpers.Tc.Uint32(), "tc_stats_map")
```

Comment:
Looks like they were all my issues :(

Reply:
Lol too much switching back from rust and semicolons 🤣